Tests/Functional/SingleFailingTestCase/main.swift

```diff
@@ -0,0 +1,28 @@
+// RUN: %swiftc %s -o %built_tests_dir/SingleFailingTestCase
+// RUN: %built_tests_dir/SingleFailingTestCase > %test_output || true
+// RUN: %xctest_checker %test_output %s
+// CHECK: Test Case 'SingleFailingTestCase.test_fails' started.
+// CHECK: .*/Tests/Functional/SingleFailingTestCase/main.swift:24: error: SingleFailingTestCase.test_fails : XCTAssertTrue failed - 
```

Comment:
I rebased on top of #22; the tests verify the new output from @briancroom's updates.